vue/components/ui/molecules/form-input/form-input.vue

```diff
@@ -50,6 +69,15 @@
     line-height: 0px;
 }
 
+.form-input.form-input-inline .content {
+    flex: 1;

```

Comment:
Add a min-width here as well such that both label and input can shrink at the same time